sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/ParseDriver.scala

```diff
@@ -125,13 +125,13 @@ abstract class AbstractSqlParser extends ParserInterface with Logging {
 /**
  * Concrete SQL parser for Catalyst-only SQL statements.
  */
-class CatalystSqlParser(conf: SQLConf) extends AbstractSqlParser {
+class CatalystSqlParser(conf: SQLConf) extends AbstractSqlParser(conf) {
   val astBuilder = new AstBuilder(conf)
 }
 
 /** For test-only. */
-object CatalystSqlParser extends AbstractSqlParser {
-  val astBuilder = new AstBuilder(new SQLConf())
+object CatalystSqlParser extends AbstractSqlParser(SQLConf.get) {
+  val astBuilder = new AstBuilder(SQLConf.get)
```

Comment:
@AngersZhuuuu . We should not change this line because we don't have SPARK-27117 in `branch-2.4.`

Could you confirm this, @cloud-fan ?

Reply:
> @AngersZhuuuu . We should not change this line because we don't have SPARK-27117 in `branch-2.4.`
> 
> Could you confirm this, @cloud-fan ?

Seems he made this change with the same reason as me. I think it's ok. Am I right? @cloud-fan

Reply:
`the same reason`? SPARK-27117 is not aiming `thread safe`.
- https://issues.apache.org/jira/browse/SPARK-27117

Reply:
Oh, you mean 'it's to pass the UT`?

Reply:
> Oh, do you mean 'it's to pass the UT`?

Yes, since `withSQLConf` change `SQLConf.get`. 
When he add this pr, we haven't  add ansi in `ParserDriver`,but use it in in `ASTBuilder`.

Reply:
Got it. Thanks. I'm reviewing again.

Reply:
> Got it. Thanks. I'm reviewing again.

Since test method changed, it's really confusing. hhh